### PR TITLE
feat: 🎸 reorg consulting section, add getting help section, add xsede

### LINKED ISSUE #130: Consulting Section suggestions
1. Combine HPC and Visualization into Advanced Research Computing. I'm not sure whether the name "Advanced Research Computing" is dead.  Maybe just a one page on consulting.  

2. Remove Discourse from this section - maybe have a "getting help" section?

### LINKED ISSUE #137: Add XSEDE info

### LINKED ISSUE #151: [FEATURE]Contact Us Section
**Is your feature request related to a problem? Please describe.**
Ex. I'm always frustrated when [...]
We have multiple ways of being contacted. it-help, discourse, ccv-share.slack, stronghold-help, jupyterhub-help. ccv@brown.edu. 

**Describe the solution you'd like**

I think @hkershaw-brown mentioned it in another issue, but it may be a good idea to have a getting-help or getting-intouch section that clearly outlines the available mediums and which one to use when.

### PR DESCRIPTION
**Pull request by fernandogelin**
closes #130, closes #151, closes #137 

### Pull Request
---

#### PR Summary
Reorganize the Consulting section. Add Getting Help Section. Add Slack link in the footer.
Add XSEDE Info.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [x] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [ ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

File: content/english/services/consulting/advanced-research-computing.md [hunk 1]
```diff
@@ -20,6 +20,40 @@ The breath of methods with which we have expertise is broad—spanning anything
 
 CCV's research software engineers work with researchers to develop scientific software that is essential to conduct their research. This includes, for example, the development of tools and libraries for working with data, building workflows and infrastructure, and web application development. Our team of research software engineers also work with researchers to help take existing software and optimize it for performance and efficiency.
 
+## High-Preformance Computing
```
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md]: typo - `Performance`

File: content/english/services/consulting/advanced-research-computing.md [hunk 2]
```diff
@@ -20,6 +20,40 @@ The breath of methods with which we have expertise is broad—spanning anything
...
+
+CCV's team of computational scientists and software engineers help researchers get the most out of Brown's high-performance computing (HPC) resources. They provide a range of services, including:
```
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md]: no one has the title of `computational scientist` - should this be `data scientist`?

File: content/english/services/consulting/advanced-research-computing.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 title: Advanced Research Computing
 
 tagTitle: Advanced Research Computing - Center for Computation and Visualization
-tagDescription: Find support for researchers seeking help with statistical modeling, machine learning, data mining, data visualization, and software engineering
+tagDescription: Support for researchers seeking help with statistical modeling, machine learning, data mining, data visualization, computational biology, high-performance computing, and software engineering
 
 date: 2019-01-24 20:20:43 +0000
 category: Consulting
-lead: Support for researchers seeking help with statistical modeling, machine learning, data mining, data visualization, and software engineering
+lead: Support for researchers seeking help with statistical modeling, machine learning, data mining, data visualization, computational biology, high-performance computing, and software engineering.
```
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md]: the other leads don't have a period - looks out of place

File: content/english/services/consulting/advanced-research-computing.md [hunk 4]
```diff
@@ -20,6 +20,40 @@ The breath of methods with which we have expertise is broad—spanning anything
...
+* Code profiling to identify bottlenecks and optimize performance
+* Assistance using Oscar, Brown's research computing cluster
+* Assistance accessing computational resources made available through [XSEDE](https://www.xsede.org)
+
+If you would like more information on CCV's HPC services, please contact [support@ccv.brown.edu](mailto:support@ccv.brown.edu).
+
+## Visualization
+
+CCV's team of graphics software engineers provides a wide range of services to assist researchers at Brown with visualization projects, including:
+ - Expertise with graphics software and packages
```
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md:38]: be consistent in use of `-` or `*` for bulleted lists

File: content/english/services/consulting/advanced-research-computing.md [hunk 5]
```diff
@@ -20,6 +20,40 @@ The breath of methods with which we have expertise is broad—spanning anything
...
+ - Familiarity with a wide range of cutting-edge visualization hardware
+ - Custom software development  
+ - Experience implementing advanced visualiztion technology in labs and work environments across campus
+
+If you would like more information on CCV's visualization services, please contact [support@ccv.brown.edu](mailto:support@ccv.brown.edu).
+
+## Computational Biology
+
+The Computational Biology Core provides assistance in experimental design and data processing pipelines for high-throughput datasets generated in individual projects, particularly for DNA/RNA sequencing data. Our assistance falls broadly into the categories outlined below.
```
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md]: Avoid use of `Our` - `CBC's` instead.

File: content/english/services/consulting/advanced-research-computing.md (hunk 1 above)
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md]: these new sections seem pretty repetitive with `our teams` - what is the intended difference?

File: content/english/services/consulting/deskpro.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+title: "CCV Helpdesk"
```
Comment (mcmcgrath13) [inline content/english/services/consulting/deskpro.md:2]: all of the `getting help` cards `tagDescriptions` use full sentences, whereas the other sections cards do not - make consistent

File: content/english/services/consulting/discourse.md [hunk 7]
```diff
@@ -6,10 +6,16 @@ tagDescription: Got Questions? Discover the Q&A platform for researchers, facili
...
-icon: support
+icon: question
+direct_link:
+  - text: Brown Space <i class="fab fa-discourse"></i>
+    url: https://ask.cyberinfrastructure.org/c/brown-research-computing
+  - text: General ask.ci <i class="fab fa-discourse"></i>
+    url: https://ask.ci
 ---
 
 A shared community resource, providing a discussion forum to share skills, knowledge and interests through ongoing conversation.
```
Comment (mcmcgrath13) [inline content/english/services/consulting/discourse.md]: This seems like it should be the lead, and the lead should be this.

File: content/english/services/consulting/office_hours.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Office Hours"
+
+tagTitle: Office Hours
+tagDescription: CCV Share is a collection of Slack channels where CCV staff and the community will be available to discuss your questions!
```
Comment (mcmcgrath13) [inline content/english/services/consulting/office_hours.md]: this `tagDescription` doesn't match the title/lead

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: content/english/services/consulting/advanced-research-computing.md (hunk 3 above)
Comment (fernandogelin) [reply content/english/services/consulting/advanced-research-computing.md]: @mcmcgrath13 do you have a suggestion?

File: content/english/services/consulting/advanced-research-computing.md (hunk 1 above)
Comment (fernandogelin) [reply content/english/services/consulting/advanced-research-computing.md]: @paulstey what do you think?

File: content/english/services/consulting/deskpro.md (hunk 6 above)
Comment (fernandogelin) [reply content/english/services/consulting/deskpro.md:2]: this doesn't matter for getting help. That can be removed.

File: content/english/services/consulting/advanced-research-computing.md (hunk 3 above)
Comment (mcmcgrath13) [reply content/english/services/consulting/advanced-research-computing.md]: Remove the period

File: content/english/services/consulting/advanced-research-computing.md (hunk 2 above)
Comment (fernandogelin) [reply content/english/services/consulting/advanced-research-computing.md]: research software engineers only I would say

File: content/english/services/consulting/deskpro.md (hunk 6 above)
Comment (mcmcgrath13) [reply content/english/services/consulting/deskpro.md:2]: is `tagDescription` not what's used in the cards in the services page? whatever that part is is what's out of stylistic sync with the other carsd.

File: content/english/services/consulting/advanced-research-computing.md (hunk 3 above)
Comment (fernandogelin) [reply content/english/services/consulting/advanced-research-computing.md]: oh nm I misread..

File: content/english/services/consulting/deskpro.md (hunk 6 above)
Comment (fernandogelin) [reply content/english/services/consulting/deskpro.md:2]: this should be a separate issue. It looks like there's a duplication of content, the tags are only being used in twitter/og tags.

File: content/english/services/consulting/advanced-research-computing.md (hunk 1 above)
Comment (paulstey) [reply content/english/services/consulting/advanced-research-computing.md]: I think there certainly is overlap, but certainly some differences. If it helps, I'd be more than happy to write a more general introductory section, and then integrate all the existing content in to the description of the teams. What do you all think?

@mcmcgrath13 @fernandogelin @mirestrepo

Comment (fernandogelin) [reply content/english/services/consulting/advanced-research-computing.md]: that would be great @paulstey

**Review by mttunca — APPROVED** [review]

File: content/english/services/consulting/advanced-research-computing.md (hunk 1 above)
Comment (paulstey) [reply content/english/services/consulting/advanced-research-computing.md]: Okay, great! I'll take this, @fernandogelin  and @mcmcgrath13!!

Comment (fernandogelin) [reply content/english/services/consulting/advanced-research-computing.md]: @paulstey did you have time to work on this?

File: content/english/services/consulting/advanced-research-computing.md [hunk 9]
```diff
@@ -20,6 +20,40 @@ The breath of methods with which we have expertise is broad—spanning anything
```
Comment (mcmcgrath13) [inline content/english/services/consulting/advanced-research-computing.md:20]: on line 17 it should say `The breadth...` instead of `breath`

**Pull request merged**